etc/tokens.json

```diff
@@ -484,5 +484,14 @@
     "decimals": 8,
     "logo": "https://github.saturn.network/etc/logo/lct_2.png",
     "forum": "https://forum.saturn.network/t/placeholder-for-something-new-d/3157" 
+  },
+  {
+    "name": "Candy Classic",
+    "address": "0x0b6f824fab88281ea8bd883bdfc1f586743f92fa",
+    "symbol": "CANDY",
+    "decimals": 6,
+    "logo": "https://github.saturn.network/etc/logo/candy final.png",
```

Comment:
you named you image `Candy Final.png` but here you wrote `candy final.png`. Uppercase/lowercase matters!

And space ` ` is a bad character to use in URLs. I suggest you rename your image as `candy_final.png` and reupload, and change this line to reflect this change.